internal/binutils/binutils_test.go

```diff
@@ -189,14 +191,39 @@ func skipUnlessDarwinAmd64(t *testing.T) {
 }
 
 func testDisasm(t *testing.T, intelSyntax bool) {
+	if runtime.GOOS == "darwin" {
+		// get objdump version and skip test if < 9.0
```

Comment:
nit: get -> Get (proper sentences for comments).

Reply:
done.